frontend/src/scenes/insights/aggregationAxisFormat.ts

```diff
@@ -1,17 +1,21 @@
-import { LemonSelectOptions } from 'lib/components/LemonSelect'
-import { capitalizeFirstLetter, compactNumber, humanFriendlyDuration, percentage } from 'lib/utils'
+import { LemonSelectOption } from 'lib/components/LemonSelect'
+import { compactNumber, humanFriendlyDuration, percentage } from 'lib/utils'
 import { ChartDisplayType } from '~/types'
 
 const formats = ['numeric', 'duration', 'percentage'] as const
 export type AggregationAxisFormat = typeof formats[number]
 
-export const isAggregationAxisFormat = (candidate: unknown): candidate is AggregationAxisFormat =>
-    formats.includes(candidate as AggregationAxisFormat)
-
-export const aggregationAxisFormatSelectOptions = formats.reduce((target, format) => {
-    target[format as string] = { label: capitalizeFirstLetter(format as string) }
-    return target
-}, {} as LemonSelectOptions)
+export const aggregationAxisFormatSelectOptions: Record<AggregationAxisFormat, LemonSelectOption> = {
+    numeric: {
+        label: 'None',
+    },
+    duration: {
+        label: 'Duration (s)',
```

Comment:
Yep, I see... so we might need to add 'Duration (ms)' as an option too. Because not every input will be seconds.